flytekit/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 import flytekit.plugins  # noqa: F401
 
-__version__ = "0.12.4"
+__version__ = "0.12.5"
```

Comment:
nit question, do we have to change the version every change?

Reply:
yes